src/Glpi/Form/AccessControl/FormAccessControl.php

```diff
@@ -70,7 +70,11 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
             return false;
         }
 
-        return self::createTabEntry(self::getTypeName());
+        $form_access_mananger = FormAccessControlManager::getInstance();
+        return self::createTabEntry(
+            self::getTypeName(),
+            count($form_access_mananger->getActiveAccessControlsForForm($item)),
```

Comment:
You should count only if `$_SESSION['glpishow_count_on_tabs']`.

```suggestion
            $_SESSION['glpishow_count_on_tabs'] ? count($form_access_mananger->getActiveAccessControlsForForm($item)) : 0,
```

Reply:
I keep forgetting this option must be checked into each items type...

Could be a new backlog issue but it would be more simple if the method was split in two "getTabName" and "countItems" method.
Then the internal framework could decide by itself to call "countItems" if the session allow it, thus centralizing this check and avoid having to repeat it in every items.